### PR TITLE
l337-redis: Upgrade redis crate to 0.21

### PR DESCRIPTION
**Pull request by keithnoguchi**

### CODE REVIEW

File: l337-redis/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ description = "l337 manager for redis"
 l337 = { version = "0.9", path = ".." }
 futures = "0.3"
 tokio = "1"
-redis = { version = "0.20", features = ["aio", "tokio-comp"] }
+redis = { version = "0.21", features = ["aio", "tokio-comp"] }
```
Comment (lilymara-onesignal) [inline l337-redis/Cargo.toml:13]: Crate version will need a bump, and add this to changelog please

**Review by lilymara-onesignal — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "l337"
-version = "0.9.0"
+version = "0.10.0"
```
Comment (lilymara-onesignal) [inline Cargo.toml:3]: If `l337` version is updated then the `l337` dependencies in the impl crates also need to change

**Review by lilymara-onesignal — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (keithnoguchi) [reply Cargo.toml:3]: I'll squash those commits when I merge.

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,10 +2,10 @@ version: 2
 jobs:
   build:
     docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/rust:1.48
-      - image: redis:4.0.1
+      - image: circleci/rust:1.57
```
Comment (lilymara-onesignal) [inline .circleci/config.yml:5]: Minimum supported rust version changing should also be reported in the changelog

**Review by lilymara-onesignal — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (keithnoguchi) [reply .circleci/config.yml:5]: Good point!  Thanks Lily!

Comment (keithnoguchi) [reply .circleci/config.yml]: I believe it's ready for the review now.  Please review, @lilymara-onesignal.

**Review by lilymara-onesignal — APPROVED** [review]

**Pull request merged**